product_docs/docs/pgd/5/rel_notes/index.mdx

```diff
@@ -15,6 +16,7 @@ that introduced the feature.
 
 | Release Date  |   EDB Postgres Distributed   | BDR extension | PGD CLI | PGD Proxy |
 | ------------- | ---------------------------- | ------------- | ------- | --------- |
+| 2023 May 16   | [5.1.0](pgd_5.1.0_rel_notes) | 5.1.0         | 5.1.0   | 5.1.0     |
 | 2023 Mar 21   | [5.0.1](pgd_5.0.1_rel_notes) | 5.0.0         | 5.0.1   | 5.0.1     |
```

Comment:
We have a future date for 5.0.1? Sorry, not tracking releases closely, but it seemed odd.